src/Compilers/Core/Portable/Syntax/ChildSyntaxList.Enumerator.cs

```diff
@@ -18,12 +18,14 @@ public struct Enumerator
             private SyntaxNode? _node;
             private int _count;
             private int _childIndex;
+            SlotData _slotData;
```

Comment:
private?

Reply:
note: while this makes the enumerator a little larger, it seems well worth it.

Reply:
private-d!